megazords/full/Cargo.toml

```diff
@@ -11,11 +11,12 @@ crate-type = ["cdylib"]
 [dependencies]
 fxa-client = { path = "../../components/fxa-client" }
 logins = { path = "../../components/logins" }
+tabs = { path = "../../components/tabs/" }
```

Comment:
huh - why is this necessary and/or how did it work without it?

Reply:
I think because `sync_manager_ffi` depended on tabs so it was pulled in transitively.  But then `sync_manager` also depends on tabs, so I'm not exactly sure what's going on.  In any case, it seems good to explicitly list all the components that are in the megazord.